### ISSUE TITLE
The volume down hotkey is not working

### ISSUE THREAD

**Issue by KirosC** [mumble-voip/mumble#2803]
The volume up hotkey is working well.  However the volume down hotkey cannot change the volume all the time.

**Comment by davidebeatrici** [comment]
Operating system? Mumble version? Which hotkeys are you using?

**Comment by KirosC** [comment]
Window 7.
1.2.19
I tried several keys on the keyboard and mouse, all are not working.

**Comment by mkrautz** [comment]
Just a quick question: is it possible this is triggered by whether Mumble is in the foreground?

Does it work every time if Mumble isn't the active program?
Does it fail every time if Mumble is the active program?

Thanks.

**Comment by KirosC** [comment]
This happens no matter Mumble is in foregroud and background
It's weird, sometimes the hotkey works for once or twice, then the hotkey make no change again.
Am I the only one encountering this issue?

**Comment by mkrautz** [comment]
I can reproduce.

For some reason, I initially tested with volume up.

**Comment by mkrautz** [comment]
Marking as up-for-grabs... Maybe it's the rounding of the floating point math in the up/down handlers that is wrong:

https://github.com/mumble-voip/mumble/blob/32030ed/src/mumble/MainWindow.cpp#L2467-L2481

**Comment by savv01** [comment]
Grabbing this. Will report back with results.

**Comment by rhotenm92** [comment]
We looked into this and were able to recreate on Windows 10. I noticed that the hot keys would not work only when the volume was set to 110%. Similarly, the volume up hotkey would not work when the volume was 10%. Our first theory is that it is caused by the inherent unpredictability of floats. In essence when g.s.fVolume = 1.1f, it is read as 1.1000001, which after being multiplied by 10 = 11.000001. the ceiling function of this produces 12, and thus the volume is set to 1.1f. 

We are currently working on implementing a fix and will create a pull request once we solve. Why would this only occur on Windows? Any input would be appreciated.
